README.md

```diff
@@ -1,24 +1,38 @@
-# Sensu Plugin Skeleton
+## Sensu-Plugins-skel
```

Comment:
Should update with the actual plugin name this was just a copy/paste left over from generating this repo from our skel.